src/modules/0.0.3/aws_route53_hosted_zones/index.ts

```diff
@@ -163,8 +177,9 @@ export const AwsRoute53HostedZoneModule: Module = new Module({
             if (!newEntity) return;
```

Comment:
Maybe here should be a `continue`?

Reply:
Oh, yes, you're right. I missed that one.

Reply:
```suggestion
            if (!newEntity) continue;
```